prac_10/wiki.py

```diff
@@ -3,8 +3,10 @@
 user_input = input("Search:")
 while user_input != '':
     try:
-        search = wikipedia.summary(user_input)
-        print(search)
+        search = wikipedia.page(user_input)
+        print(search.title)
+        print(search.summary)
+        print(search.url)
```

Comment:
My only suggestion would be to put your print statements in one:
print("{}\n {}\n {}\n".format()